CIP-0137/README.md

```diff
@@ -151,19 +151,19 @@ stateDiagram-v2
 23  messageList = [ * message ]
 24  messageSizeInBytes = word32
 25  kesSignature = bstr
-26  operationalCertificate = bstr
-27  kesPeriod = word32
-28  blockNumber = word32
-29  ttl = word16
+26  kesPeriod = word32
```

Comment:
In `kes-agent` library `KESPeriod` is `Word` rather than `Word32`, [ref](https://github.com/input-output-hk/kes-agent/blob/2a9ac9c017aa33c4f64b052c2d91babbbed842d3/kes-agent/src/Cardano/KESAgent/KES/OCert.hs#L43).

Reply:
@coot, this means that we need to encode it as `word64`. WDYT?